tutorials/dapt-curation/code/requirements.txt

```diff
@@ -3,5 +3,7 @@ arxiv-downloader
 cchardet
 nltk==3.8.1
 poppler-utils
+qgrid
+tesseract-ocr
```

Comment:
We recently discussed this, let's pin version here so that we don't see breakage with subsequent releases cc @ayushdg @ryantwolf